src/sentry/api/endpoints/team_time_to_resolution.py

```diff
@@ -0,0 +1,58 @@
+from collections import defaultdict
+from datetime import timedelta
+
+from django.db.models import Avg, F
+from django.db.models.functions import TruncDay
+from rest_framework.response import Response
+
+from sentry.api.base import EnvironmentMixin
+from sentry.api.bases.team import TeamEndpoint
+from sentry.api.utils import get_date_range_from_params
+from sentry.models import GroupHistory, GroupHistoryStatus, Project
+
+
+class TeamTimeToResolutionEndpoint(TeamEndpoint, EnvironmentMixin):
+    def get(self, request, team):
+        """
+        Return a a time bucketed list of mean group resolution times for a given team.
+        """
+        project_list = Project.objects.get_for_team_ids(team_ids=[team.id])
+        start, end = get_date_range_from_params(request.GET)
+        end = end.replace(hour=23, minute=59, second=59, microsecond=999999)
+        start = start.replace(hour=0, minute=0, second=0, microsecond=0) + timedelta(days=1)
```

Comment:
If you only need the date components you could do `start.date()` and `end.date()` to get the date from the datetime.

Reply:
Thanks, thats simpler than zero filling...I ended up just using the date as the key so didn't end up trying to feed it through the better default encoder (which I had issues with because I wanted to form a dict and the key could not be a datetime...so I just imported it and called it on the dict key which felt less the ideal, but it didn't like the fact that start is a "FakeDate".